src/dotnet/APIView/APIViewWeb/Managers/ReviewManager.cs

```diff
@@ -303,12 +309,16 @@ public async Task ToggleApprovalAsync(ClaimsPrincipal user, string id, string re
             {
                 //Revert approval
                 revision.Approvers.Remove(userId);
+                await _signalRHubContext.Clients.Group(user.Identity.Name).SendAsync("ReceiveApprovalSelf", review.ReviewId, revisionId, false);
+                await _signalRHubContext.Clients.All.SendAsync("ReceiveApproval", review.ReviewId, revisionId, userId, false);
```

Comment:
These two lines are almost identical of lines 320-321. This can be deduplicated with a variable for type of message and flag set here and call SendAsync only once.